src/wp-admin/includes/dashboard.php

```diff
@@ -2005,54 +2005,29 @@ function wp_welcome_panel() {
 	<h2><?php _e( 'Welcome to WordPress!' ); ?></h2>
 	<p class="about-description"><?php _e( 'We&#8217;ve assembled some links to get you started:' ); ?></p>
 	<div class="welcome-panel-column-container">
-	<div class="welcome-panel-column">
-		<?php if ( $customize_url ) : ?>
-			<h3><?php _e( 'Get Started' ); ?></h3>
-			<a class="button button-primary button-hero load-customize hide-if-no-customize" href="<?php echo $customize_url; ?>"><?php _e( 'Customize Your Site' ); ?></a>
-		<?php endif; ?>
-		<a class="button button-primary button-hero hide-if-customize" href="<?php echo esc_url( admin_url( 'themes.php' ) ); ?>"><?php _e( 'Customize Your Site' ); ?></a>
-		<?php if ( current_user_can( 'install_themes' ) || ( current_user_can( 'switch_themes' ) && count( wp_get_themes( array( 'allowed' => true ) ) ) > 1 ) ) : ?>
-			<?php $themes_link = $can_customize && ! $is_block_theme ? add_query_arg( 'autofocus[panel]', 'themes', admin_url( 'customize.php' ) ) : admin_url( 'themes.php' ); ?>
-			<p class="hide-if-no-customize">
-				<?php
-					/* translators: %s: URL to Themes panel in Customizer or Themes screen. */
-					printf( __( 'or, <a href="%s">change your theme completely</a>' ), $themes_link );
-				?>
-			</p>
-		<?php endif; ?>
+	<div class="welcome-panel-column is-welcome-panel-patterns">
+		<div class="welcome-panel-image"></div>
+		<h3><?php _e( 'Author rich content with blocks and patterns' ); ?></h3>
+		<p><?php _e( 'Use a block pattern to create new pages in a flash, or add some pizazz to your next post.' ); ?></p>
+		<div class="welcome-panel-button-row">
+			<?php printf( '<a href="%s" class="button button-secondary">' . __( 'New post' ) . '</a>', admin_url( 'post-new.php' ) ); ?>
+			<?php printf( '<a href="%s" class="button button-secondary">' . __( 'New page' ) . '</a>', admin_url( 'post-new.php?post_type=page' ) ); ?>
```

Comment:
I think let's avoid `printf` here just so that we're not mixing different ways of concatenating strings together. It makes it a little bit easier for future programmers to spot issues to do with not escaping the HTML, which are very common and can be nasty.

Speaking of which, we also need to escape the URL using `esc_url`.

```suggestion
			<a href="<?php echo esc_url( admin_url( 'post-new.php' ) ); ?>" class="button button-secondary"><?php _e( 'New post' ); ?></a>
			<a href="<?php echo esc_url( admin_url( 'post-new.php?post_type=page' ) ); ?>" class="button button-secondary"><?php _e( 'New page' ); ?></a>
```

Reply:
Done ✅